app/models/event.rb

```diff
@@ -83,8 +83,9 @@ def self.format_date(type)
 
   #scope
   scope :recent_events, -> {order(created_at: :DESC).limit(12)}
-  scope :featured_events, -> {order(created_at: :DESC).limit(2)}
-  scope :popular_events, -> {order(created_at: :DESC).limit(3)}
+  scope :featured_events, -> {order(created_at: :DESC).limit(12)}
+  scope :popular_events, -> {find_by_sql("Select events.*, COUNT(bookings.event_id) AS num from events INNER JOIN bookings WHERE bookings.event_id = events.id GROUP BY events.id ORDER BY num")}
+  scope :upcoming_events, -> { where("start_date >= ?", Time.now).order("start_date ASC").limit(12) }
```

Comment:
Line is too long. [101/80]<br>Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.